rpcd/playbooks/roles/rpc_maas/defaults/main.yml

```diff
@@ -269,7 +269,7 @@ innodb_row_lock_time_avg_critical_threshold: 10000
 # cinder-volumes volume group thresholds
 cinder_volumes_vg_warning_threshold: 80.0
 cinder_volumes_vg_critical_threshold: 90.0
-cinder_vg_name: "{% if cinder_backends is defined %}{{cinder_backends['lvm']['volume_group']}}{% endif %}"
+cinder_vg_name: "{{ cinder_backends['lvm']['volume_group'] | default('') }}"
```

Comment:
I don't recommend to use empty volume group name by default, which will be confused when maas cinder warning message comes up. AIO scenario has one default name called 'cinder-volumes'. Reference here: https://github.com/openstack/openstack-ansible/blob/master/etc/openstack_deploy/conf.d/cinder.yml.aio#L13